logical/plugin/pb/backend.proto

```diff
@@ -209,9 +209,7 @@ message LeaseOptions {
 	
 	bool renewable = 2;
 
-	int64 increment = 3;
```

Comment:
Consider marking these deprecated instead: https://developers.google.com/protocol-buffers/docs/proto3#options

Reply:
This is fixed in https://github.com/hashicorp/vault/pull/4230/commits/790b9c21976818ce81adfb964a5cd44a7d9e8d1e.